superset-frontend/src/datasource/DatasourceEditor.jsx

```diff
@@ -616,7 +616,12 @@ class DatasourceEditor extends React.PureComponent {
                 'values from the table. Typically the intent would be to limit the scan ' +
                 'by applying a relative time filter on a partitioned or indexed time-related field.',
             )}
-            control={<TextControl controlId="fetch_values_predicate" />}
+            control={
+              <TextAreaControl
+                language="sql"
+                controlId="fetch_values_predicate"
+              />
```

Comment:
```suggestion
              <TextAreaControl
                language="sql"
                controlId="fetch_values_predicate"
                minLines={5}
                maxLines={20}
              />
```

Reply:
that's looks much better.. thanks @michael-s-molina!